portlet/exoadmin/src/main/java/org/exoplatform/organization/webui/component/UIAccountEditInputSet.java

```diff
@@ -147,7 +147,7 @@ public boolean save(OrganizationService service) throws Exception {
         Query query = new Query();
         String email = getUIStringInput("email").getValue();
         query.setEmail(email);
-        if (!oldEmail.equals(email) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
+        if (oldEmail != null && !oldEmail.equals(email) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
```

Comment:
It means we don't check anymore for new email existence if the user did not have an email, so we could end up having 2 users with the same email, no ?